### ISSUE TITLE
[Enhancements] Add "All" namespace view 

### ISSUE THREAD

**Issue by aperullo** [vmware-tanzu/octant#219]
Add the ability to see all resources in the cluster across all namespaces like the default kubernetes dashboard has: 

![image](https://user-images.githubusercontent.com/18688190/63796445-160bba00-c8d4-11e9-8a87-6348555b5825.png)

This is a nice way to get a good view of the cluster over all. For example, If I am working to get something like the Rook operator working, which many of my other services across namespaces rely on; it would be nice to see which ones came up successfully on one screen, without checking individual namespaces.

**Comment by bryanl** [comment]
Hi, thanks for the request. This would be an interesting addition, but we would need to change how Octant works to make this possible. I'll post back in here when I have some ideas on how we could accomplish this.

**Comment by aaroniscode** [comment]
I've become used to typing `kubectl get <resource> -A` when troubleshooting issues and would love to be able to see that view expressed visually in Octant.

**Comment by gadelkareem** [comment]
Any updates on this?

**Comment by gillg** [comment]
Hi, like @aaroniscode for debugging, I often list all pods accross all namespaces in a first approach, to have a global view of problems / anormal restarts / etc.

For information... My basic checklist is :
```
# List nodes with taints could be very usefull to understant cluster state issues
kubectl get nodes -o custom-columns=NAME:.metadata.name,TAINTS:.spec.taints,CAPACITY:.status.allocatable.pods

# Cluster status
kubectl cluster-info

# Get cluster events
kubectl get event -A --field-selector 'type!=Normal' --watch=true

# List all pods
kubectl get pods -A -o wide
kubectl get pods -A --field-selector 'status.phase!=Running' -o wide

# Get events of not running pods
for pod in $(kubectl get pods -A --field-selector 'status.phase!=Running' -o jsonpath="{.items[*].metadata.name}"); do echo; echo $pod; kubectl get event -A --field-selector involvedObject.name=$pod; done
```

**Comment by juliohm1978** [comment]
This is the first thing I missed when trying octant. Here's me hoping it gets added. I'm pretty sure lots of people would consider it a basic feature.

**Comment by dyipon** [comment]
+1

**Comment by petermicuch** [comment]
Exactly. This was the first thing that I missed when getting overview of all pods running in my system. Something like **Lens** or **Kubernetic** or **Headlamp** has. Actually, pretty much any k8s UI client does have that (even command line k9s).

Otherwise Octant is nice tool, especially in Dark theme :-), with its dependency graph and port forwarding capability.

**Comment by brgsousa** [comment]
+1
